STRegex.podspec

```
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   s.version      = %x(git describe --tags --abbrev=0).chomp.sub(/^v/, '')
   s.summary      = "A Swift µframework providing an NSRegularExpression-backed Regex type"
   s.homepage     = "https://github.com/sharplet/Regex"
-  s.license      = "LICENSE.txt"
+  s.license      = "MIT"
```

Thanks for the idea, happy to expose some more structured information about the license type. I think this might be the correct definition, based on the [Podspec Syntax Reference](https://guides.cocoapods.org/syntax/podspec.html#license):
```suggestion
  s.license      = { type: "MIT", file: "LICENSE.txt" }
```